demo/scripts/grasp.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python
```

Comment:
You added `demo/scripts/grasp.py`, `demo/scripts/pick.py`, `notes.org`, `pick+place.org` and `test.py` to the repo in this commit.

I guess the last three were committed by accident?

Reply:
Oops. That was an accident indeed. If you don't mind, I would like to rewrite history :wink:

Reply:
You know my opinion on force pushes.
This has been upstream for 4 days and we apparently have an active user base these days.
If you consider your org-mode files private information, feel free to go ahead.
Otherwise I would simply add a removal-commit.